src/runner/nunit.runner.uwp/project.json

```diff
@@ -1,12 +1,12 @@
 ﻿{
   "dependencies": {
     "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
-    "NUnit": "[3.8.1]",
+    "NUnit": "3.9.0",
     "PCLStorage": "1.0.2",
-    "Xamarin.Forms": "1.5.0.6447"
+    "Xamarin.Forms": "2.5.0.122203"
   },
   "frameworks": {
-    "uap10.0": {}
+    "uap10.0.10240": {}
```

Comment:
Was this autogenerated? Not sure why we'd want to specify a particular UWP version here.

Reply:
I didn't touch that, it must have got auto-generated. Removing...